src/containers/Classroom.jsx

```diff
@@ -34,6 +36,7 @@ export default class Classroom extends Component {
 
 Classroom.propTypes = {
```

Comment:
Probably better to specify exactly what the propTypes are, using [`.shape()`](https://facebook.github.io/react/docs/reusable-components.html)